libkbfs/favorites.go

```diff
@@ -125,15 +211,18 @@ func (f *Favorites) handleReq(req *favReq) (err error) {
 	//  * The user asked us to refresh
 	//  * We haven't fetched it before
 	//  * It's stale
-	if req.refresh || f.cache == nil || time.Now().After(f.cacheExpireTime) {
+	if req.refresh || f.cache == nil || time.Now().After(f.
+		cacheExpireTime) && !req.clear {
+
+		// load cache from server
 		folders, err := kbpki.FavoriteList(req.ctx)
 		if err != nil {
 			if req.refresh {
 				// if we're supposed to refresh the cache and it's not
-				// working, try again in a few minutes.
-				in5Minutes := time.Now().Add(5 * time.Minute)
-				if in5Minutes.After(f.cacheExpireTime) {
-					f.cacheExpireTime = in5Minutes
+				// working, try again in a minute.
+				in1Minute := time.Now().Add(time.Minute)
```

Comment:
Why the change here?  Mostly just curious.

Reply:
I don't remember. I think one minute seems more reasonable to me if the reason for failure is that the computer is starting up and network doesn't exist yet.